client/components/Modals/Modal.jsx

```diff
@@ -7,6 +7,11 @@ class Modal extends React.Component {
   constructor(props) {
     super(props);
   }
+  // When the modals unmount allow scrolling once more
+  componentWillUnmount() {
```

Comment:
I never tried to use will unmount, I should look into this

Reply:
Yeah definetely.